client/src/app.js

```diff
@@ -24,28 +28,56 @@ if (!__DEV__) {
 }
 
 console.log(`Using GraphQL endpoint: ${GRAPHQL_ENDPOINT}`);
-const networkInterface = createNetworkInterface({ uri: GRAPHQL_ENDPOINT });
+
 let store;
 
 export const bugsnag = !__DEV__ ? new BugSnagClient() : undefined;
 export const pushManager = new PushManager();
 
-// middleware for requests
-networkInterface.use([
-  {
-    applyMiddleware(req, next) {
-      if (!req.options.headers) {
-        req.options.headers = {};
-      }
-      // get the authentication token from local storage if it exists
-      const { token } = store.getState().auth;
-      if (token) {
-        req.options.headers.authorization = `Bearer ${token}`;
-      }
-      next();
+const httpLink = new HttpLink({ uri: GRAPHQL_ENDPOINT });
+
+// replaces networkInterface.use(applyMiddleware...)
+const authMiddleware = new ApolloLink((operation, forward) => {
+  // add the authorization to the headers
+  operation.setContext({
+    headers: {
+      authorization: store.getState().auth || null,
```

Comment:
We should align the web with the mobile app (see below with what is working for web) - I'm open to using either:

```
const authMiddleware = new ApolloLink((operation, forward) => {
  if (store.getState().auth.token) {
    operation.setContext({
      headers: {
        authorization: store.getState().auth.token ? `Bearer ${store.getState().auth.token}` : null,
      },
    });
  }

  return forward(operation);
});
```